daemon/src/main/java/org/mvndaemon/mvnd/daemon/Server.java

```diff
@@ -430,7 +430,7 @@ private void cancelNow() {
     private void handle(DaemonConnection connection, BuildRequest buildRequest) {
         updateState(Busy);
         final BlockingQueue<Message> sendQueue = new PriorityBlockingQueue<>(64,
-                Comparator.comparingInt(this::getClassOrder).thenComparingLong(Message::timestamp));
```

Comment:
If you really wanted to remove getClassOrder from the comparator, then the getClassOrder() method can be removed. 
And we perhaps also do not need a PriorityBlockingQueue anymore.